### PR TITLE
Finished Project

### PR DESCRIPTION
**Pull request by Dandush03**
Finished mimicking a Apple like page from 2004
Testing Bugs With Stickler

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -1,10 +1,168 @@
...
-    
-  </body>
+
+<head>
+  <title></title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body>
+  <section class="contents">
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<main>``` instead of ```<section class="contents">```

File: index.html [hunk 2]
```diff
@@ -1,10 +1,168 @@
...
+    <div class="s4">
+      <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/supplier-responsibility/">
+        <img
+          src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg"
+          width="364" height="200" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+      </a>
+    </div>
+  </section>
+</body>
+<footer>
```
Comment (addod19) [inline index.html]: Let us put the ```<footer>``` inside the ```<body>```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @Dandush03 , you have done a great job. 👍 Meanwhile we have to changes some few things to get the project approved. Kindly check the comments below:

- Kindly fix the errors returned by [w3validator](https://validator.w3.org)
markup

- Kindly fix the errors returned by [cssvalidator](https://jigsaw.w3.org/css-validator/)
k2

- The nav should have nine different buttons.Kindly refer here
but

Kindly fix these changes and submit another code review. 😄 
Happy coding 💻 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Comment by pelzolga123** [comment]
## Hi Daniel!  
### Your project looks :+1: amazing!
Just one step left before this project can be approved:
* [x] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by Dandush03** [comment]
> ## Status: Require changes 

Hey!, 

Thanks for your time and patience

I couldn't find the missing button
Screenshot from 2020-01-22 18-18-03

as you can see from the image above, they are the same. and I was looking in the code requirement I didn't find that one if you could please send me the link I'll appreciate 

besides this issue, I fixed all the other bugs I had in my code

regards.

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
Go ahead and merge to master.
_Goodluck in your next project_ *Happy coding*

**Pull request merged**